src/api/apiFunctions.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+export function postNewComment(article_id, commentData) {
+    return axios.post(`https://backend-project-c921.onrender.com/api/articles/${article_id}/comments`, commentData)
```

Comment:
consider returning just the relevant key from the response.data object rather than the whole response object